docs/realm.js

```diff
@@ -426,6 +426,7 @@ class Realm {
  *    what fits in memory, but it is not persistent and will be removed when the last instance
  *    is closed. This option is incompatible with `sync`.
  * @property {boolean} [readOnly=false] - Specifies if this Realm should be opened as read-only.
+ * @property {boolean} [cache=true] - Cache Realm instances. 
```

Comment:
Couldn't this use an explanation as to when to cache and when not to?

Reply:
Were we caching previously?

Reply:
No, this is something we specifically do not expose publicly.

Reply:
There's a lint error here that should be fixed.

Reply:
I think you should fix/reference count close rather than expose cache.

Reply:
It's a workaround until we have the time and resources to fix said issue.

Reply:
Is the plan then to make a breaking change later to remove the option once this real issue is fixed?

As a minimum, there has to be a descriptive parameter description.